incubator/couchbase-operator/templates/admission-cluster-role-binding.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if and .Values.rbac.create  .Values.deployments.admissionController -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "admission-controller.fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
can you update the labels to use the new ones? if you use the latest helm when you create the project it already add the new lables, but here is for reference https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata